### PR TITLE
Update Customer.php

### PR DESCRIPTION
**Pull request by mbensoussan**

### CODE REVIEW

File: app/code/community/PH2M/Gdpr/Model/Override/Customer/Customer.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ class PH2M_Gdpr_Model_Override_Customer_Customer extends Mage_Customer_Model_Cus
      */
     public function validate()
     {
-        $errors = [];
+        $errors = array();
```
Comment (VincentMarmiesse) [inline app/code/community/PH2M/Gdpr/Model/Override/Customer/Customer.php]: Why don't you let the `[]` syntax?

Comment (mbensoussan) [reply app/code/community/PH2M/Gdpr/Model/Override/Customer/Customer.php]: i don't know the difference, but magento core files make this change in the lastest version.
I think it's for php7.

Comment (FredericMartinez) [reply app/code/community/PH2M/Gdpr/Model/Override/Customer/Customer.php]: You may try to use PHP 7 on your M1 projects.
- 1.9.4.0 = Officiel Magento Release accept PHP 7.2 (and of course 7.1 & 7.0)
- 1.9.* = With Inchoo PHP7 module: https://github.com/Inchoo/Inchoo_PHP7 (thousand of M1-PHP7 projects in production)

Benefits:
- 50% speedup
- [PSR2 standard coding + Array short syntax](https://github.com/PH2M/GDPR/blob/master/.php_cs.dist#L12)

Comment (mbensoussan) [reply app/code/community/PH2M/Gdpr/Model/Override/Customer/Customer.php]: i use it ... in my case.
and the lane is better for php 7.2 $errors = array();

Comment (ioweb-gr) [reply app/code/community/PH2M/Gdpr/Model/Override/Customer/Customer.php]: I don't see anything on php site about removing the shorthand syntax for array(). It doesn't seem to throw any syntax errors. There are some changes when you use empty index keys in arrays that will now throw fatal errors but nothing about the shorthand. 

The shorthand syntax should work on any version above 5.4 so maybe it has something to do with coding standards that magento uses?

Comment (bastienlm) [reply app/code/community/PH2M/Gdpr/Model/Override/Customer/Customer.php]: for module coherence (and PSR2), we keep [ ] (used 7 times in module)

**Comment by bastienlm** [comment]
Thanks for your contribution :+1:

**Pull request merged**